global.json

```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.x"
+    "version": "6.0.0",
+    "rollForward": "latestFeature"
```

Comment:
ISSUE: We should probably add `"allowPrerelease": false`.

The [documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#globaljson-schema) says that it will default to `true`.


> If you don't set this value explicitly, the default value depends on whether you're running from Visual Studio:
If you're not in Visual Studio, the default value is true.

Reply:
Thanks for catching this, @MoFtZ!
I have added it in the next commit.